README.md

```diff
@@ -310,23 +310,38 @@ The NeMo Framework can be installed in a variety of ways, depending on
 your needs. Depending on the domain, you may find one of the following
 installation methods more suitable.
 
-- Conda / Pip - Refer to [Conda](#conda) and [Pip](#pip) for
-  installation instructions.
+- [Conda / Pip](#conda--pip): Install NeMo-Framework with native Pip into a virtual environment.
+  - Perfect to explore NeMo on any supported platform.
   - This is the recommended method for ASR and TTS domains.
-  - When using a Nvidia PyTorch container as the base, this is the
-      recommended method for all domains.
-- Docker Containers - Refer to [Docker containers](#docker-containers)
-  for installation instructions.
-  - NeMo Framework container -
-      [nvcr.io/nvidia/nemo:24.05]{.title-ref}
-- LLMs and MMs Dependencies - Refer to [LLMs and MMs
-    Dependencies](#install-llms-and-mms-dependencies) for installation
-    instructions.
+  - Limited feature-completeness for other domains.
+- [NGC PyTorch container](#ngc-pytorch-container): Install NeMo-Framework from source with feature-completeness into a highly optimized container.
+  - Perfect for customers that seek flexibility.
```

Comment:
Again let's use another word besides "perfect". Also flexibility is a bit generic. Maybe just say what it is. For users that want to install from source in a highly optimized container.